e2e/react/src/react.test.ts

```diff
@@ -401,3 +402,52 @@ describe('React Applications', () => {
     }
   }
 });
+
+describe('--style option', () => {
+  beforeAll(() => newProject());
+
+  it.each`
+    style
+    ${'css'}
+    ${'scss'}
+    ${'less'}
+    ${'styl'}
+  `('should support global and css modules', ({ style }) => {
+    const appName = uniq('app');
+    runCLI(
+      `generate @nrwl/react:app ${appName} --style=${style} --no-interactive`
+    );
+
+    // make sure stylePreprocessorOptions works
+    updateWorkspaceConfig((workspace) => {
+      workspace.projects[
+        appName
+      ].targets.build.options.stylePreprocessorOptions = {
+        includePaths: ['libs/shared/lib'],
+      };
+      return workspace;
+    });
+    updateFile(
+      `apps/${appName}/src/styles.${style}`,
+      `@import 'base.${style}';`
+    );
+    updateFile(
+      `apps/${appName}/src/app/app.module.${style}`,
+      (s) => `@import 'base.${style}';\n${s}`
```

Comment:
There's still some inconsistency in that for instance in Angular you can import things like `@import 'base'` without specifying the file extension.

That works in `*.module.scss` files, but not in the global `styles.scss`

Reply:
Need to investigate further. It should be support by sass automatically so not sure why it's not working.